LPD-41466 Create playwright tests for "View" and "Permissions" permissions

thektan opened the pull request:
https://liferay.atlassian.net/browse/LPD-41466

Mainly for @kresimir-coko and @chestofwonders: An update on the branch I shared in the group chat. The most notable part is the `setupUserRoleAndLoginAsUser` function that now allows different actions to be selected. An example use would be:

```
await setupUserRoleAndLoginAsUser({
	apiHelpers,
	dataSetResourcePermissions: [
		{
			actions: ['Permissions', 'View'],
			name: 'Data Set',
		},
	],
	page,
	roleDefinePermissionsPage,
	rolePage,
	rolesPage,
});
```

All that's missing is the cleanup steps to delete the created role and user which we could do something similar to the `test.afterEach` in collecting the created role/user ids or ercs.

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-28.liferay.com/job/test-portal-source-format/6920/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/6920/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2c0f2ffb6d183d8450238c156894088a564f9d39">2c0f2ffb6d183d8450238c156894088a564f9d39</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/thektan/liferay-portal/tree/LPD-23542_LPD-41466_CreatePlaywrightTestsForViewAndPermissionsPermissions">LPD-23542_LPD-41466_CreatePlaywrightTestsForViewAndPermissionsPermissions</a><br/>Branch GIT ID: <a href="https://github.com/thektan/liferay-portal/commit/38df1e096a6d1b86203ba38ff61f76c059543484">38df1e096a6d1b86203ba38ff61f76c059543484</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-source-format/6920/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/6920/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-source-format/6920/">test-portal-source-format#6920</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/6920/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4558">liferay-frontend#4558</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/70105224">[master] ci:test:sf - thektan &gt; liferay-frontend - PR#4558 - 2024-11-15[16:04:49]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>70105224<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/publish-testray-report/25715/">publish-testray-report#25715</a><br/></div></div>

dsanz commented on modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts:
```diff
@@ -150,6 +163,131 @@ async function assertTableRowsCount(page, rowsCount) {
+				{
+					actionIds: ['VIEW_CONTROL_PANEL'],
+					primaryKey: companyId,
+					resourceName: '90',
```
why 90?

thektan replied:
Apparently it's the key for Portal, not sure how that number was chosen though:

https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/PortletKeys.java#L132

thektan commented:
Added @kresimir-coko's commit on top from https://github.com/liferay-frontend/liferay-portal/pull/4568/commits

I had a lot of issues with the tests not properly logging in as the admin user so I've added additional step to login for the moment: https://github.com/liferay-frontend/liferay-portal/pull/4558/commits/e721487949ec25caee1de85368f05ab4a46c5463#diff-2091b1c9d07194c1e29c22731a1a9944965c70916e131384a9f9dd09bf209830R302-R319

I'm still unsure why it happens (maybe only locally) but when I locally test, if a subsequent test runs after a test that logs in as a new user and back to admin, the new test doesn't log back in as admin. This change fixes it but definitely doesn't look great to keep. I'll need to look more into it.

thektan commented:
ci:test:playwright

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>playwright</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/11855/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11855/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 85 out of 110 jobs passed in 1 hour 28 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/38072c8dd20cb75551606707642c4b7d77a3ca5a">38072c8dd20cb75551606707642c4b7d77a3ca5a</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/98457fab147b2d80667dfea44b015932e0ea67f3">98457fab147b2d80667dfea44b015932e0ea67f3</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/74699032">EE Development Acceptance (master) - 1035 - 2024-11-21[12:42:29]</a></p><details><summary><strong>ci:test:playwright - 85 out of 110 jobs PASSED</strong></summary><div><h4>25 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/11855/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>playwright-compile<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033496/">playwright-compile/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984061/"><strike>playwright-js-tomcat90-mysql57/0/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130172/">playwright-js-tomcat90-mysql57/1/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984064/">playwright-js-tomcat90-mysql57/2/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130177/"><strike>playwright-js-tomcat90-mysql57/3/0</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093160/"><strike>playwright-js-tomcat90-mysql57/4/0</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790557/">playwright-js-tomcat90-mysql57/5/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261475/">playwright-js-tomcat90-mysql57/6/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261477/">playwright-js-tomcat90-mysql57/7/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130169/">playwright-js-tomcat90-mysql57/8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142183/">playwright-js-tomcat90-mysql57/9/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790554/">playwright-js-tomcat90-mysql57/10/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093164/">playwright-js-tomcat90-mysql57/11/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027841/">playwright-js-tomcat90-mysql57/12/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790562/">playwright-js-tomcat90-mysql57/13/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261469/">playwright-js-tomcat90-mysql57/14/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1028501/">playwright-js-tomcat90-mysql57/15/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033499/">playwright-js-tomcat90-mysql57/16/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027843/">playwright-js-tomcat90-mysql57/17/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033493/">playwright-js-tomcat90-mysql57/18/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261472/">playwright-js-tomcat90-mysql57/19/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1028505/"><strike>playwright-js-tomcat90-mysql57/20/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130167/">playwright-js-tomcat90-mysql57/21/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142181/">playwright-js-tomcat90-mysql57/22/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984063/">playwright-js-tomcat90-mysql57/23/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027842/">playwright-js-tomcat90-mysql57/24/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027832/">playwright-js-tomcat90-mysql57/25/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790555/">playwright-js-tomcat90-mysql57/26/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1028502/"><strike>playwright-js-tomcat90-mysql57/27/0</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146164/"><strike>playwright-js-tomcat90-mysql57/27/1</strike></a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984065/">playwright-js-tomcat90-mysql57/28/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790550/">playwright-js-tomcat90-mysql57/29/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261473/">playwright-js-tomcat90-mysql57/30/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/963610/"><strike>playwright-js-tomcat90-mysql57/31/0</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238931/">playwright-js-tomcat90-mysql57/32/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984059/">playwright-js-tomcat90-mysql57/33/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984062/">playwright-js-tomcat90-mysql57/34/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790560/"><strike>playwright-js-tomcat90-mysql57/35/0</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1028506/">playwright-js-tomcat90-mysql57/36/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027835/">playwright-js-tomcat90-mysql57/37/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027838/">playwright-js-tomcat90-mysql57/38/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130175/">playwright-js-tomcat90-mysql57/39/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130168/">playwright-js-tomcat90-mysql57/40/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261471/">playwright-js-tomcat90-mysql57/41/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261474/">playwright-js-tomcat90-mysql57/42/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238930/">playwright-js-tomcat90-mysql57/43/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261467/"><strike>playwright-js-tomcat90-mysql57/44/0</strike></a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984060/">playwright-js-tomcat90-mysql57/45/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142184/">playwright-js-tomcat90-mysql57/46/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261476/"><strike>playwright-js-tomcat90-mysql57/47/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033500/">playwright-js-tomcat90-mysql57/47/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261466/">playwright-js-tomcat90-mysql57/48/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027836/">playwright-js-tomcat90-mysql57/49/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093162/">playwright-js-tomcat90-mysql57/50/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790552/">playwright-js-tomcat90-mysql57/51/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146167/">playwright-js-tomcat90-mysql57/52/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238928/">playwright-js-tomcat90-mysql57/53/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027837/"><strike>playwright-js-tomcat90-mysql57/54/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027834/">playwright-js-tomcat90-mysql57/55/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093168/"><strike>playwright-js-tomcat90-mysql57/56/0</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093166/">playwright-js-tomcat90-mysql57/57/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142182/">playwright-js-tomcat90-mysql57/58/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142179/">playwright-js-tomcat90-mysql57/59/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238929/">playwright-js-tomcat90-mysql57/60/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033497/"><strike>playwright-js-tomcat90-mysql57/61/0</strike></a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142180/">playwright-js-tomcat90-mysql57/62/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/963609/">playwright-js-tomcat90-mysql57/63/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1028507/">playwright-js-tomcat90-mysql57/63/1</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146166/">playwright-js-tomcat90-mysql57/64/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027839/">playwright-js-tomcat90-mysql57/65/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130170/"><strike>playwright-js-tomcat90-mysql57/66/0</strike></a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142188/">playwright-js-tomcat90-mysql57/67/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093167/">playwright-js-tomcat90-mysql57/68/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238927/">playwright-js-tomcat90-mysql57/69/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027833/">playwright-js-tomcat90-mysql57/70/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790561/"><strike>playwright-js-tomcat90-mysql57/71/0</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790559/">playwright-js-tomcat90-mysql57/72/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033495/">playwright-js-tomcat90-mysql57/73/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261468/"><strike>playwright-js-tomcat90-mysql57/74/0</strike></a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984057/">playwright-js-tomcat90-mysql57/75/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142185/"><strike>playwright-js-tomcat90-mysql57/76/0</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146165/"><strike>playwright-js-tomcat90-mysql57/76/1</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790556/">playwright-js-tomcat90-mysql57/77/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984058/"><strike>playwright-js-tomcat90-mysql57/78/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033501/">playwright-js-tomcat90-mysql57/79/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146163/">playwright-js-tomcat90-mysql57/80/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093161/"><strike>playwright-js-tomcat90-mysql57/81/0</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/721985/">playwright-js-tomcat90-mysql57/82/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1028504/">playwright-js-tomcat90-mysql57/83/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130173/">playwright-js-tomcat90-mysql57/84/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033494/">playwright-js-tomcat90-mysql57/85/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146168/">playwright-js-tomcat90-mysql57/86/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142186/">playwright-js-tomcat90-mysql57/87/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790558/">playwright-js-tomcat90-mysql57/88/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261470/">playwright-js-tomcat90-mysql57/89/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130176/">playwright-js-tomcat90-mysql57/90/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093165/">playwright-js-tomcat90-mysql57/91/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130171/">playwright-js-tomcat90-mysql57/92/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033492/">playwright-js-tomcat90-mysql57/93/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/721986/"><strike>playwright-js-tomcat90-mysql57/94/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1033498/">playwright-js-tomcat90-mysql57/95/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1093163/">playwright-js-tomcat90-mysql57/96/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790553/">playwright-js-tomcat90-mysql57/97/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142187/">playwright-js-tomcat90-mysql57/98/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1028503/"><strike>playwright-js-tomcat90-mysql57/99/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1130174/"><strike>playwright-js-tomcat90-mysql57/100/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1027840/">playwright-js-tomcat90-mysql57/100/1</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/963611/"><strike>playwright-js-tomcat90-mysql57/101/0</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/790551/">playwright-js-tomcat90-mysql57/102/0</a></li></ul></li></ul></div><details><summary><strong>85 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11855/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146164//consoleText">playwright-js-tomcat90-mysql57/27/1</a> - <a href="https://storage.cloud.google.com/testray-results/2024-11/test-1-40/test-portal-acceptance-pullrequest(master)/11855/playwright-js-tomcat90-mysql57/27/1/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146164/testReport/object-web_rootModels.spec/ts">object-web/rootModels.spec.ts</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146164//testReport/object-web_rootModels.spec/ts/Manage_root_models_elements_through_Objects_Admin___cannot_delete_an_object_definition_with_inheritance_enabled_on_its_relationship">Manage root models elements through Objects Admin › cannot delete an object definition with inheritance enabled on its relationship</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261468//consoleText">playwright-js-tomcat90-mysql57/74/0</a> - <a href="https://storage.cloud.google.com/testray-results/2024-11/test-1-40/test-portal-acceptance-pullrequest(master)/11855/playwright-js-tomcat90-mysql57/74/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261468/testReport/layout-admin-web_pageWithWorkflow.spec/ts">layout-admin-web/pageWithWorkflow.spec.ts</a></summary><div><ol><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261468//testReport/layout-admin-web_pageWithWorkflow.spec/ts/Page_author_cannot_edit_again_after_submit">Page author cannot edit again after submit</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142185//consoleText">playwright-js-tomcat90-mysql57/76/0</a> - <a href="https://storage.cloud.google.com/testray-results/2024-11/test-1-40/test-portal-acceptance-pullrequest(master)/11855/playwright-js-tomcat90-mysql57/76/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142185/testReport/commerce_commerce-order-content-web_commerceLayouts.spec/ts">commerce/commerce-order-content-web/commerceLayouts.spec.ts</a></summary><div><ol><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1142185//testReport/commerce_commerce-order-content-web_commerceLayouts.spec/ts/LPD_33809_Edit_Payment_Method_in_Open_Order_Details">LPD-33809 Edit Payment Method in Open Order Details</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146165//consoleText">playwright-js-tomcat90-mysql57/76/1</a> - <a href="https://storage.cloud.google.com/testray-results/2024-11/test-1-40/test-portal-acceptance-pullrequest(master)/11855/playwright-js-tomcat90-mysql57/76/1/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146165/testReport/commerce_commerce-order-content-web_return.spec/ts">commerce/commerce-order-content-web/return.spec.ts</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146165//testReport/commerce_commerce-order-content-web_return.spec/ts/LPD_32515_Returns_widget_displays_amount_fields_with_correct_currency_pattern">LPD-32515 Returns widget displays amount fields with correct currency pattern</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146165//testReport/commerce_commerce-order-content-web_return.spec/ts/LPD_32519_Warning_message_before_submitting_a_return_should_not_be_shown_once_the_return_has_been_submitted">LPD-32519 Warning message before submitting a return should not be shown once the return has been submitted</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146165/testReport/commerce_commerce-order-content-web_commerceOrderDisplayPageTemplates.spec/ts">commerce/commerce-order-content-web/commerceOrderDisplayPageTemplates.spec.ts</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1146165//testReport/commerce_commerce-order-content-web_commerceOrderDisplayPageTemplates.spec/ts/LPD_30855_Can_map_order_item_detailed_information">LPD-30855 Can map order item detailed information</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984058//consoleText">playwright-js-tomcat90-mysql57/78/0</a> - <a href="https://storage.cloud.google.com/testray-results/2024-11/test-1-40/test-portal-acceptance-pullrequest(master)/11855/playwright-js-tomcat90-mysql57/78/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984058/testReport/osb-faro-web_onboarding.spec/ts">osb-faro-web/onboarding.spec.ts</a></summary><div><ol><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/984058//testReport/osb-faro-web_onboarding.spec/ts/Modal_is_displayed_on_first_access_to_a_new_workspace">Modal is displayed on first access to a new workspace</a></li></ol></div></details></li></ol></div></li><li>...</li></ol></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11855/bundles/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a></li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/11855/">test-portal-acceptance-pullrequest(master)#11855</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11855/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/4558">liferay-frontend#4558</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/74854453">[master] ci:test:playwright - thektan &gt; liferay-frontend - PR#4558 - 2024-11-21[18:26:35]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>74854453<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/25833/">publish-testray-report#25833</a><br/></div></div>

thektan commented:
For the login issue, still unsure why this happens, but when the next test runs and the browser opens, the 2nd browser doesn't refresh as logged in at this step: https://github.com/liferay-frontend/liferay-portal/blob/master/modules/test/playwright/fixtures/loginTest.ts#L62

So so far my best option is having this `beforeEach` https://github.com/liferay-frontend/liferay-portal/pull/4558/commits/e721487949ec25caee1de85368f05ab4a46c5463#diff-2091b1c9d07194c1e29c22731a1a9944965c70916e131384a9f9dd09bf209830R302-R319 🫤

dsanz commented:
@juanjofgliferay is adding these commits into the big PR we're preparing, so I'm closing this one
